kotlin-android/app/src/main/java/io/realm/examples/objectserver/TaskActivity.kt

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2019 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.examples.objectserver
+
+import android.app.AlertDialog
+import android.content.DialogInterface
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.view.Menu
+import android.view.MenuItem
+import android.widget.EditText
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.ItemTouchHelper
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.floatingactionbutton.FloatingActionButton
+import io.realm.Realm
+import io.realm.RealmResults
+import io.realm.RealmUser
+import io.realm.SyncConfiguration
+import io.realm.examples.objectserver.model.ListAdapter
+import io.realm.examples.objectserver.model.Project
+import io.realm.examples.objectserver.model.SwipeToDeleteCallback
+import io.realm.examples.objectserver.model.Task
+import io.realm.kotlin.syncSession
+import io.realm.kotlin.where
+import io.realm.log.RealmLog
+
+
+class TaskActivity : AppCompatActivity() {
+    private var realm: Realm? = null
+    private var recyclerView: RecyclerView? = null
+    private var adapter: ListAdapter? = null
+    private var user: RealmUser? = null
+    private var fab: FloatingActionButton? = null
```

Comment:
Many of these could be declared [lateinit](https://kotlinlang.org/docs/reference/properties.html#late-initialized-properties-and-variables) to reduce optional-unwrapping sludge